lib/configuration/configuration.js

```diff
@@ -119,9 +141,18 @@ class Configuration {
 
     // probotContext.config loads config from current repo or from a repo called
     // '.github' in the same organisation as a fallback. It returns the parsed YAML object.
-
+    let config = null
     const configPath = process.env.CONFIG_PATH ? process.env.CONFIG_PATH : 'mergeable.yml'
-    const config = await context.probotContext.config(configPath)
+    if (configCache !== null) {
+      Configuration.updateCache(context)
+      config = configCache.get(`${repo.owner}/${repo.repo}`)
+      if (config === undefined) {
```

Comment:
Since we have loadash in the codebase and you're using it on L97, I propose we stay consistent here and use `_.isUndefined`